src/dotnet/Chat.UI.Blazor/Components/PlaceMenu/PlaceMenu.razor

```diff
@@ -50,10 +50,10 @@ else if (_canLeave) {
     private NavbarUI NavbarUI => Hub.NavbarUI;
 
     protected override async Task OnParametersSetAsync() {
-        if (Arguments is not [ var placeSid ])
+        if (Arguments is not [ var PlaceId ])
```

Comment:
Here we can keep 'placeSid' or use 'sPlaceId' like in other places.

Reply:
done